pvlib/pvsystem.py

```diff
@@ -1373,6 +1379,54 @@ def get_iam(self, aoi, iam_model='physical'):
             raise ValueError(model + ' is not a valid IAM model')
 
 
+class FixedMount:
```

Comment:
This is pretty heavy-weight for something that is basically a glorified tuple. Could we do something like this?

```python
class FixedMount(namedtuple('FixedMount', ['surface_tilt', 'surface_azimuth'], defaults=[0, 180])):
    def get_orientation(self, solar_zenith, solar_azimuth):
        return self
```

I'm not sure how that would go with a `Mount` ABC, just throwing it out there.

Reply:
It is pretty silly.  I'm also thinking about consistency of docs pages across the Mount classes.  I went with ABC for now, but am happy to change if there's a good way to reduce the clunk.

Reply:
Instead of a namedtuple, what about a dataclass? Less boilerplate than a regular class, types for free, good performance. There's now precedent for them in pvlib (`ModelChainResult`). From a little bit of googling, it looks like dataclasses work with ABCs so long as both the abstract and the implementation classes are decorated with `@dataclass`.